components/cd-card/cd-card.html

```diff
@@ -0,0 +1,156 @@
+<link rel=stylesheet href="../cd-button/cd-button.css" type="text/css" media=screen>
+<link rel=stylesheet href="../cd-read-more/cd-read-more.css" type="text/css" media=screen>
+<link rel=stylesheet href="cd-card.css" type="text/css" media=screen>
+
+<div class="cd-layout-container">
+  <div class="cd-container cd-grid cd-grid-3-col">
+
+    <article role="article" class="cd-card node--view-mode-card [ cd-flow ]">
+
+      <div class="cd-card__container [ cd-flow ]">
+
+        <div class="cd-card__title">
+          <a href="#" rel="bookmark">Sed dapibus lectus sit amet adipiscing egestas mauris est viverra nibh</a>
+        </div>
+
+        <div class="cd-card__description node__content">
+          <p>Sed dapibus, lectus sit amet adipiscing egestas, mauris est viverra nibh (iaculis pretium sem orci aliquet mauris). Nulla auctor eleifend turpis consequat pharetra. Nulla vel magna sit amet dui lobortis commodo vitae vel nulla sit amet ante hendrerit tempus. Donec a congue leo? </p>
+        </div>
+
+      </div>
+
+      <div class="cd-card__footer">
+        <a href="#" class="cd-card__link cd-read-more">
+          Learn more
+          <svg class="cd-icon cd-icon--arrow-right" aria-hidden="true" focusable="false" width="16" height="16">
+            <use xlink:href="#cd-icon--arrow-right"></use>
+          </svg>
+        </a>
+      </div>
+
+    </article>
+
+
+    <article role="article" class="cd-card node--view-mode-card [ cd-flow ]">
+
+      <div class="cd-card__image">
+        <img src="https://via.placeholder.com/410x210" alt="card image example">
+      </div>
+
+      <div class="cd-card__container [ cd-flow ]">
+
+        <div class="cd-card__title">
+          <h3><a href="#" rel="bookmark">Sed dapibus lectus sit amet adipiscing egestas mauris est viverra nibh</a></h3>
+        </div>
+
+        <div class="cd-card__date">
+          30 Jul 2019
+        </div>
+
+        <div class="cd-card__description node__content">
+          <p>Sed dapibus, lectus sit amet adipiscing egestas, mauris est viverra nibh (iaculis pretium sem orci aliquet mauris). Nulla auctor eleifend turpis consequat pharetra. Nulla vel magna sit amet dui lobortis commodo vitae vel nulla sit amet ante hendrerit tempus. Donec a congue leo? </p>
+        </div>
+
+      </div>
+
+      <div class="cd-card__footer cd-align--right">
+        <a href="#" class="cd-card__link cd-read-more">
+          Learn more
+          <svg class="cd-icon cd-icon--arrow-right" aria-hidden="true" focusable="false" width="16" height="16">
+            <use xlink:href="#cd-icon--arrow-right"></use>
+          </svg>
+        </a>
+      </div>
+
+    </article>
+
+    <article role="article" class="cd-card node--view-mode-card [ cd-flow ]">
+
+      <div class="cd-card__image">
+        <img src="https://via.placeholder.com/410x210" alt="card image example">
+      </div>
+
+      <div class="cd-card__container cd-align--center [ cd-flow ]">
```

Comment:
the `.cd-align--*` classes are another utility that has an ambiguous origin. My assumption is that I will find a component  called `cd-XXX` if I see a class name starting with that.

In this case it might be as easy as adjusting the dashes:

```
.cd--align-center {}
```

The double-dash after CD tells me this is a top-level CD thing, and `align-center` is the name of that thing. It's technically a mis-use of the double-dash; I guess a true namespace for the utilities like `.cd-util__align-center` would be the "correct" solution but that's a lot to type and look at, and presents its own problems since the top-level `cd-util` is nowhere to be found.

Reply:
Thanks, I take your points. Not sure how to proceed exactly but it's worth rethinking.

Reply:
All my comments with `cd-utilities` might be "beginner" problems that go away with familiarity.

However that file contained multiple things that I had to go "figure out" as I reviewed the code. I suppose all the other components re so nicely scoped and contained, so it was jarring to have to go hunt down classes elsewhere. You spoiled me!